packages/fxa-auth-server/fxa-oauth-server/lib/jwt.js

```diff
@@ -52,7 +52,6 @@ exports.verify = async function verify(jwt, options = {}) {
 
   return jwtverify(jwt, getKey, {
     algorithms: options.algorithms || [SIGNING_ALG],
-    complete: true,
```

Comment:
This was the suprising knock-on dependency change I had to make after updating shrinkwrap.

Reply:
Ref: https://github.com/auth0/node-jsonwebtoken/pull/522

Reply:
Huh... Was I using a newer version of jsonwebtoken while developing? This does indeed seem like the correct behavior now.